modules/django-files/files/models.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.db import models
+
+from modules.utils import get_options
+
+MEDIA_UPLOAD_PATH = get_options("files", "MEDIA_UPLOAD_PATH")
+
+
+class FileUploaded(models.Model):
```

Comment:
Would it make more sense to call this "FileUpload"/"FileUploads" and set the media path at "mediafiles/uploads"?

Reply:
I was suggesting renaming the model